assets/input/floatingLabel.css

```diff
@@ -8,31 +8,44 @@ input.floatingLabel-input:focus {
 }
 
 input.floatingLabel-input:focus ~ .floatingLabel-label {
-  @apply text-teal
+  @apply text-teal;
```

Comment:
not sure why is ` ;` added

Reply:
My Ide decided to as it was there on all other lines I thought I would keep it for consistency